addons/crm/models/crm_lead.py

```diff
@@ -217,6 +217,18 @@ def _compute_is_automated_probability(self):
         for lead in self:
             lead.is_automated_probability = tools.float_compare(lead.probability, lead.automated_probability, 2) == 0
 
+    @api.depends(lambda self: ['tag_ids', 'stage_id', 'team_id'] + self._pls_get_safe_fields())
+    def _compute_automated_probability(self):
+        for lead in self:
+            was_automated = False
+            lead_probabilities = lead._pls_get_naive_bayes_probabilities()
+            if lead.id in lead_probabilities:
+                if(tools.float_compare(lead.probability, lead.automated_probability, 2) == 0):
+                    was_automated = True
+                lead.automated_probability = lead_probabilities[lead.id]
+                if was_automated:
+                    lead.probability = lead.automated_probability
```

Comment:
Can't write on a regular field in a compute method.
The "_compute" attribute of the probability field should also be in that method then.